packages/nocodb-sdk/src/lib/is.ts

```
@@ -0,0 +1,207 @@
+ *
+ * @param value - The value to check.
+ * @returns {boolean} - True if the value is a number, false otherwise.
+ *
+ * @example
+ * ```typescript
+ * const value = 42;
+ * console.log(ncIsNumber(value)); // true
+ * ```
+ */
+export function ncIsNumber(value: any): value is number {
+  return typeof value === 'number' && !isNaN(value)
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)